src/ngx_http_lua_util.c

```diff
@@ -371,40 +371,29 @@ ngx_http_lua_del_thread(ngx_http_request_t *r, lua_State *L,
 u_char *
 ngx_http_lua_rebase_path(ngx_pool_t *pool, u_char *src, size_t len)
 {
-    u_char            *p, *dst;
-
-    if (len == 0) {
+    ngx_str_t nsrc;
+    u_char *optr;
```

Comment:
Style: identifiers of local variable declarations must be aligned up (see other similar places in the existing code base).